src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -52,10 +52,23 @@
  * {@code false}.</li>
  * </ul>
  * <p>
- * The following example shows how to configure the {@code AbstractClassName} to
- * checks names, but ignore missing {@code abstract} modifiers:
+ * How to configure the {@code AbstractClassName} to use
+ * default values:
+ * </p>
+ * <pre>
+ * &lt;module name="AbstractClassName"/&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * abstract class AbstractFirstClass {} // OK
+ * abstract class SecondClass {} // violation, it should match pattern "^Abstract.+$"
+ * class AbstractThirdClass {} // violation, abstract access modifier should be present
```

Comment:
Update to `// violation, must be declared 'abstract'`

Reply:
done